### PR TITLE
Add GET user content recommendation

### PR DESCRIPTION
**Pull request by ashyablok**
Add GET content user recommendation

Example Usage:

``` go
key := "APIKEY"
client := lytics.NewLytics(key, nil)
ql := "FILTER AND (url LIKE \"www.example.com/*\") FROM content"

// Accepts fieldName, fieldValue, filterql (strings), limit (int), and shuffle (boolean)
recs, err := client.GetUserContentRecommendation("user_id", "USERID", ql, 3, false)
```

### CODE REVIEW

**Comment by araddon** [comment]
nice, ty!   :+1:

File: content.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+// GetUserContentRecommendation returns
+// a list of documents to recommend the user
+// based on their content affinity
+func (l *Client) GetUserContentRecommendation(fieldName, fieldVal, ql string, limit int, shuffle bool) ([]Recommendation, error) {
+  res := ApiResp{}
+  data := []Recommendation{}
+  params := map[string]string{}
+
+  if limit > -1 {
```
Comment (markhayden) [inline content.go]: assuming passing -1 disregards limit would u ever have a `0` limit?

File: content.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  *Document
+  Confidence      float64            `json:"confidence"`
+  Visited         bool               `json:"visited"`
+  VisitRate       float64            `json:"visitrate,omitempty"`
+}
+
+// GetUserContentRecommendation returns
+// a list of documents to recommend the user
+// based on their content affinity
+func (l *Client) GetUserContentRecommendation(fieldName, fieldVal, ql string, limit int, shuffle bool) ([]Recommendation, error) {
```
Comment (markhayden) [inline content.go:43]: might be worth adding some or at least one example of a valid `ql` string here.

**Comment by markhayden** [comment]
👍

**Pull request merged**